package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@p2olab/polaris-interface",
   "description": "interface definitions for polaris-frontend and polaris-backend",
-  "version": "3.0.0-alpha0.8",
+  "version": "3.0.0-alpha5",
```

Comment:
whats about that version raise?

Reply:
I'm using integers for the alpha tag (like in pimad-core). I hope thats okay.